src/pages/stacks-blockchain/testnet.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Testnet
+description: Learn about testnet phases, timelines, and features
+images:
+  large: /images/pages/testnet.svg
+  sm: /images/pages/testnet-sm.svg
+---
+
+## Introduction
+
+The Stacks 2.0 testnet is an alterative blockchain, to be used for testing. It is intended for developer experimentation and not built for production-ready integrations. The testnet is frequently reset and Stacks tokens, used as fees for operations on the testnet, can be obtained for free using a [faucet](https://www.blockstack.org/testnet/faucet).
+
+With the completion of the Stacks 2.0 blockchain, a production-ready, stable version will be released as "mainnet". While work for the mainnet is in progress, the testnet reflects the most recent implementation of the Stacks 2.0 blockchain.
+
+## Release phases
+
+The work towards mainnet is split up into four sequential phases:
+
+- ✅ **Phase 1:** is a developer local setup, mono-node, assembling SIP 001, SIP 002, SIP 004 and SIP 005. With this version, developers can not only run Stacks 2.0 on their development machines, but also write, execute, and test smart contracts.
+- ✅ **Phase 2:** is the current version of our public testnet. This testnet includes SIP 003, and will be an open-membership public network, where participants will be able to validate and participate in mining testnet blocks.
+- [ ] **Phase 3:** is the upcoming version that we're anticipating to ship in Q3 2020. This version focus on the PoX basics (miner get rewards, Stackers get BTC distributions, etc).
```

Comment:
See http://whenmainnet.co for some language on phases.

Also, I've seen the use of both 4 phases and 5 phases. Might be good to make that consistent.

Reply:
I think the testnet page shows only 4 phases as well, right? should we stick to that?

Reply:
fyi, updated language based on suggestion